public/x-chat-list.html

```diff
@@ -11,7 +12,9 @@
         <div class="avatar {{color}}" style="background-image: url({{avatar}})"></div>
         <div flex>
             <div class="username" data-nick="{{nick}}">{{nick}}</div>
-            <div class="text" data-value="{{message}}">{{message}}</div>
+            <div class="text" data-value="{{message}}">
+                <x-html-echo html="{{message}}"></x-html-echo>
```

Comment:
Please make sure that this is not vulnerable to XSS

Reply:
I think it is! I did not think that before. Will try to resolve.

Reply:
Since html is escaped in the messages, so one thing is resolved. But if someone tries to type this
```
[some text](javascript:alert('xss'))
```
in the message box, then xss attack is possible!

How do I resolve this?

Reply:
See https://github.com/showdownjs/showdown/wiki/Markdown%27s-XSS-Vulnerability-(and-how-to-mitigate-it). 

Also, ensure that are allowing restricted markdown (we don't need triple backticks, or headers in chat messages).